_posts/2022-02-01-attribution.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Attribution"
+date:   2022-02-01 
+author: Agnese Beltramo
+category: Open Science
+tags: attribution
+---
+
+<div style="text-align: justify">
+More and more often, contemporary research is conducted by a number of young and senior researchers actively collaborating and contributing to a common project, answering an increasingly multidisplinary range of questions. 
+
+To ensure all the people involved in the research process are rightfully acknolwegded, the [Contributor Roles Taxonomy, known as CRediT author statement][1] was created. The CRediT taxonomy clearlry provides definitions for a wide range of roles to describe the personal contributions that a researcher might bring to a collaborative project. It also aims at providing a definition for what authorship should mean and acknowledge, in the case of scientific research, by broadening the concept to include all intellectual and practical contributors to the final output. 
+
+For each contributor, the following basic personal information should be provided:
+- Name and Surname
+- Affiliation
+- Open Researcher and Contributor ID ([ORCiD](<https://orcid.org/>)) code
+- CRediT role (contribution provided to the research).
```

Comment:
- I think it would be useful to add a list of the roles here. 
- You can also point out that these contributor roles are used by Zenodo (are they exactly the same?)
- The Turing Way has a section on contributors and authorship which begins to discuss the ethical aspects of authorship.  I think mentioning this would be useful.

Reply:
The Zenodo contributor roles are actually different from the CRediT ones. Covering somehow the same contributions, but also expanding a bit further I think... the Zenodo roles seem to be targeted more to project management as well. So, going beyond what are the actual people that put thoughts and time in contributing to the published work, and including also more admin/managing roles. Not sure how to combine the two. I could not find a Zenodo taxonomy of its contributor roles.